src/mmw/js/src/core/views.js

```diff
@@ -59,6 +61,10 @@ var HeaderView = Marionette.ItemView.extend({
         // Defer requiring app until needed as it is not defined when
         // core.views are initialized (they are required in app.js)
         require('../app').showLoginModal();
+    },
+
+    userLogout: function() {
+        require('../app').userLogout();
```

Comment:
I think we're doing pretty well without introducing event based communication, I would like to try and just resolve the initialization dependence through restructuring somehow.  Maybe passing in App would be the simplist.